packages/flutter_tools/test/general.shard/asset_bundle_variant_test.dart

```diff
@@ -135,12 +164,6 @@ flutter:
     late final Platform platform;
     late final FileSystem fs;
 
-    String correctPathSeparators(String path) {
-      // The in-memory file system is strict about slashes on Windows being the
-      // correct way. See https://github.com/google/file.dart/issues/112.
-      return path.replaceAll('/', fs.path.separator);
-    }
-
     setUpAll(() {
```

Comment:
```suggestion
    setUp(() {
```

Reply:
Good observation. I actually thought I made this switch at some point, but apparently not 😆

Reply:
I feel that way all the time, either because: 1) I fixed something but forgot to commit; 2) I fixed something on a different branch; 3) I fixed some similar line, but forgot to apply the same fix to another function; 4) I've been staring at code too long and am imagining things.